nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -59,6 +61,28 @@ Each endpoint uses its own Azure storage account, thereby increasing message thr
 
 ![Scale out with multiple storage accounts](azure03.png "width=50%")
 
+## Securing connection strings to storage accounts for Scale Out
+
+[Securing connection strings to storage accounts](/nservicebus/azure-storage-queues/configuration.md#securing-connection-strings-to-storage-accounts) has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, securing connection string is limited to calling `.UseAccountNamesInsteadOfConnectionStrings()`. When Scaling Out is applied, securing connection strings requires using logical names of the accounts.
+Consider the following example:
+
+- two endpoints `default` and `another` using different accounts for their input queues
```

Comment:
`default` is an overloaded name :)
Would `endpoint-A` and `endpoint-B` serve the sample better?